[2022-06-12 16:30:55,733] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ingest_data.get_data_to_ingest_from_local_file_task scheduled__2022-06-12T00:23:00+00:00 [queued]>
[2022-06-12 16:30:55,763] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ingest_data.get_data_to_ingest_from_local_file_task scheduled__2022-06-12T00:23:00+00:00 [queued]>
[2022-06-12 16:30:55,763] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-12 16:30:55,764] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-12 16:30:55,764] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-12 16:30:55,814] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): get_data_to_ingest_from_local_file_task> on 2022-06-12 00:23:00+00:00
[2022-06-12 16:30:55,830] {standard_task_runner.py:52} INFO - Started process 4084 to run task
[2022-06-12 16:30:55,828] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'ingest_data', 'get_data_to_ingest_from_local_file_task', 'scheduled__2022-06-12T00:23:00+00:00', '--job-id', '439', '--raw', '--subdir', 'DAGS_FOLDER/ingestion_dag.py', '--cfg-path', '/tmp/tmpzwk9_44z', '--error-file', '/tmp/tmp31zoya1a']
[2022-06-12 16:30:55,837] {standard_task_runner.py:80} INFO - Job 439: Subtask get_data_to_ingest_from_local_file_task
[2022-06-12 16:30:55,979] {task_command.py:370} INFO - Running <TaskInstance: ingest_data.get_data_to_ingest_from_local_file_task scheduled__2022-06-12T00:23:00+00:00 [running]> on host DESKTOP-GDPK8BL.localdomain
[2022-06-12 16:30:56,329] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=ingest_data
AIRFLOW_CTX_TASK_ID=get_data_to_ingest_from_local_file_task
AIRFLOW_CTX_EXECUTION_DATE=2022-06-12T00:23:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-12T00:23:00+00:00
[2022-06-12 16:30:56,409] {python.py:173} INFO - Done. Returned value was:           AT      V       AP     RH      PE
2926   20.37  69.48  1011.04  84.31  445.81
7803   23.07  65.27  1013.24  58.38  443.42
6164   24.97  59.87  1018.70  59.94  443.70
34666   9.54  41.17  1019.79  65.61  481.18
30056  26.76  48.41  1010.53  47.38  440.63
[2022-06-12 16:30:56,540] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=ingest_data, task_id=get_data_to_ingest_from_local_file_task, execution_date=20220612T002300, start_date=20220612T143055, end_date=20220612T143056
[2022-06-12 16:30:56,613] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-12 16:30:56,826] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
